lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -124,7 +124,7 @@ private function _add_external_html_elements()
 
         $prefix = $this->_config->get('tinymce_url');
         midcom::get()->head->add_jsfile("{$prefix}/tinymce.min.js", true);
-        midcom::get()->head->add_jsfile("{$prefix}/imagetools_functions.js", true);
+        midcom::get()->head->add_jsfile("/midcom-static/midcom.helper.datamanager2/imagetools_func.js", true);
```

Comment:
you can't hardcode `/midcom-static/`, you need to use the respective constant instead